Makefile.authx

```diff
@@ -33,6 +55,7 @@ init-authx: mkdir
 	# move to a better solution so Aman made a decision to keep this hack as-is.
 	$(eval KEYCLOAK_CLIENT_ID_64=$(shell echo -n ${KEYCLOAK_CLIENT_ID} | base64))
 	@echo $(KEYCLOAK_CLIENT_ID_64) > $(DIR)/tmp/secrets/keycloak-client-local-candig-id-64
+	$(MAKE) docker-volumes
```

Comment:
You would need to run init-docker before init-authx so the docker-volumes would have already been created. Does this cause exit or are the existing volumes ignored?

Reply:
I have never seen an exit for this, the volumes are either created or skipped.  I can double check.

Reply:
Does not matter so will skip this.  The whole authx setup assumes a basic setup is already deployed.  Is that a mistake as an assumption?